cylc/rose/utilities.py

```diff
@@ -391,9 +380,9 @@ def get_cli_opts_node(opts=None, srcdir=None):
         )
 
     # Specialised treatement of optional configs.
-    if 'opts' not in newconfig:
-        newconfig['opts'] = ConfigNode()
```

Comment:
Never False

Reply:
I think you're right but it might be worth adding an explicit `set` after the config node is created because I think this is relying on assumptions about automatic variables being set in all situations. An assumption which could potentially break down the line.